pkg/notifiers/syslog/syslog.go

```diff
@@ -188,11 +188,11 @@ func alertToCEF(alert *storage.Alert, notifier *storage.Notifier) string {
 
 	severity := alertToCEFSeverityMap[alert.GetPolicy().GetSeverity()]
 
-	return getCEFHeaderWithExtension("Alert", alert.GetPolicy().GetName(), severity, makeExtensionFromPairs(extensionList))
+	return getCEFHeaderWithExtension(fmt.Sprintf("Policy.{%s}", alert.GetPolicy().GetId()), alert.GetPolicy().GetName(), severity, makeExtensionFromPairs(extensionList))
```

Comment:
I'm not insisting that the policy ID be exposed here; it just seemed like a nice enhancement. But I'm totally happy to drop this part of you don't agree.